doc/overlays.xml

```diff
@@ -8,59 +8,62 @@
 overlays. Overlays are used to add layers in the fix-point used by Nixpkgs
 to compose the set of all packages.</para>
 
+<para>Nixpkgs can be configured with a list of overlays, which are
+applied in order. This means that the order of the overlays can be significant
+if multiple layers override the same package.</para>
+
 <!--============================================================-->
 
 <section xml:id="sec-overlays-install">
-<title>Installing Overlays</title>
+<title>Installing overlays</title>
 
-<para>The set of overlays is looked for in the following places. The
-first one present is considered, and all the rest are ignored:
+<para>The list of overlays is determined as follows:
```

Comment:
I worded the list items using "otherwise" to indicate that each one only applies if the conditions from the previous ones didn't hold, but perhaps it's not clear?